File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        posc, negc = 0, 0
        for i in nums:
            if i > 0:
                posc += 1
            if i < 0:
                negc += 1
            else:
                continue
        
        return max(posc, negc)

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int n = nums.length;

        int pcount = 0;
        int ncount = 0;

        for (int num : nums) {
            if (num > 0) {
                pcount++;
            } else if(num < 0) {
                ncount++;
            }
        }

        return Math.max(pcount, ncount);
    }
}